pubsub/cloud-client/README.md

```diff
@@ -39,7 +39,7 @@ For more samples, see the samples in
 
 #### Publish messages
 ```
-  mvn exec:java -Dexec.mainClass=com.example.pubsub.PublisherExample -Dexec.args=my-topic
+  mvn exec:java -Dexec.mainClass=com.example.pubsub.PublisherExample -Dexec.args="my-topic, num-messages"
```

Comment:
What do you think of just setting `num-messages` as `5`, since the comment below says that's what it does and more clearly portrays that, or you could update the comment on line 44. Either way, up to you. :)